### PR TITLE
[windows] Catch psutil RuntimeError (which was blocking all checks)

### PR DESCRIPTION
**Pull request by degemer**
Emit warning when not able to fetch disk latency metrics.

### CODE REVIEW

File: checks/system/win32.py [hunk 1]
```diff
@@ -223,19 +223,22 @@ def check_disk_usage(self, agentConfig):
...
+        except RuntimeError:
+            self.logger.warning("Unable to fetch disk latency metrics")
 
 
     def check(self, agentConfig):
         self.check_disk_usage(agentConfig)
         self.check_disk_latency(agentConfig)
         return self.get_metrics()
-        
+
```
Comment (LeoCavaille) [inline checks/system/win32.py]: :cake:

File: checks/system/win32.py [hunk 2]
```diff
@@ -223,19 +223,22 @@ def check_disk_usage(self, agentConfig):
...
-            self.save_sample("system.disk.read_time_pct", read_time_pct, device_name=disk_name) 
-            self.save_sample("system.disk.write_time_pct", write_time_pct, device_name=disk_name)
+        try:
+            disk_io_counters = psutil.disk_io_counters(True)
+            for disk_name, disk in disk_io_counters.iteritems():
+                read_time_pct = disk.read_time * 100.0 / 1000.0 # x100 to have it as a percentage, /1000 as psutil returns the value in ms
+                write_time_pct = disk.write_time * 100.0 / 1000.0 # x100 to have it as a percentage, /1000 as psutil returns the value in ms
+                self.save_sample("system.disk.read_time_pct", read_time_pct, device_name=disk_name)
+                self.save_sample("system.disk.write_time_pct", write_time_pct, device_name=disk_name)
+        except RuntimeError:
```
Comment (LeoCavaille) [inline checks/system/win32.py]: Should we catch more stuff? Can your confirm `disk_io_counters` can only raise `RuntimeError`?

Comment (degemer) [inline checks/system/win32.py]: I didn't think about this, but I just checked the source code, and it's the only exception which is raised inside this function.

**Comment by LeoCavaille** [comment]
Thanks!

**Pull request merged**